build-scripts/fetch-binaries.sh

```diff
@@ -41,5 +41,14 @@ echo $KUBE_VERSION > $KUBE_SNAP_BINS/version
     mv istio-${ISTIO_ERSION}/install/kubernetes/helm/istio/templates/crds.yaml ./istio-yaml/
     mv istio-${ISTIO_ERSION}/install/kubernetes/istio-demo-auth.yaml ./istio-yaml/
     mv istio-${ISTIO_ERSION}/install/kubernetes/istio-demo.yaml ./istio-yaml/
+    
+    #Linkerd on Linux    
+    echo "Fetching linkerd binary."
```

Comment:
Can we move the downloading of the linkerd binary in the wrapper (`microk8s-resources/wrappers/microk8s-linkerd.wrapper`). The wrapper would check that the binary is available and if not it would try to download it in $SNAP_DATA/bin/.

The reason for this ask is that downloading and packaging binaries inside the MicroK8s snap does not scale. The snap will grow in size too much too quickly. One of the strong points of MicroK8s is that it deploys fast and this deployment time is dominated by the time needed to download the snap from the snap store. Addons is a convenient extension point. We welcome everyone to write addon scripts but we should not force everyone to pay the penalty of downloading binaries that may never be used.

I know that this is not what we do in the case of istio. We have to change the istio wrapper too to comply with the approach above (https://github.com/ubuntu/microk8s/issues/363).